.circleci/config.yml

```diff
@@ -62,7 +62,8 @@ jobs:
               REACT_APP_LOGOUT_URL=${REACT_APP_LOGOUT_URL}
               REACT_APP_GRAPHQL_SERVER=${REACT_APP_GRAPHQL_SERVER}
               REACT_APP_SENTRY_FE_DSN=${SENTRY_FE_DSN}
-              REACT_APP_SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}
+              REACT_APP_OLD_APP_BASE_URL=${OLD_APP_BASE_URL}
+              SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}
```

Comment:
Probably overlooked during merge: this has to be `REACT_APP_SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}`